content/english/jsappybird/1-Activity/add-jump-sound.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Add Sound While the Bird is Jumping"
+date: 2020-07-29T13:24:17-07:00
+draft: false
+weight: 10
+---
+
+Let's add sounds to the bird while it is jumping
+
+Add the below command in the file [File: create-game.js]
+    jumpSound.play();
+
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `jumpSound.play();`.
+
+![alt text](../../img/jump_sound.png "image to add sound to the bird")
+
+In your console you should see a Jsappy bird jumping with sound after you press **run**:
+
+** Bird making Jump sound while its Jumping**
```

Comment:
What's this line doing?

Reply:
this was a comment ,as I cannot show output with sound.
I will update to reflect as comment